pedpy/methods/method_utils.py

```diff
@@ -545,6 +549,35 @@ def compute_crossing_frames(
         traj_data (pandas.DataFrame): trajectory data
         measurement_line (MeasurementLine): measurement line which is crossed
 
+    Returns:
+        DataFrame containing the columns 'id', 'frame', where 'frame' is
+        the frame where the measurement line is crossed.
+    """
+    return _compute_crossing_frames(
+        traj_data=traj_data,
+        measurement_line=measurement_line,
+        count_on_line=False,
+    )
+
+
+def _compute_crossing_frames(
+    *,
+    traj_data: TrajectoryData,
+    measurement_line: MeasurementLine,
+    count_on_line: bool,
+) -> pd.DataFrame:
+    """Compute the frames at the pedestrians pass the measurement line.
```

Comment:
Compute the frames at which pedestrians pass the measurement line.

If count_on_line is set to True, the crossing frame is the one where the pedestrian touches the line. Otherwise, it is the frame where the pedestrian crosses the line without stopping on it.

Reply:
Changed